File: ts-sdk/src/sdk/models/shared/userdetail.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class UserDetail extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=email" })
  email: string;

  @SpeakeasyMetadata({ data: "json, name=full_name" })
  fullName: string;

  @SpeakeasyMetadata({ data: "json, name=password" })
  password: string;
}

File: user-api/python-sdk/src/epilot/models/shared/userdetail.py
import dataclasses
from dataclasses_json import dataclass_json
from epilot import utils


@dataclass_json
@dataclasses.dataclass
class UserDetail:
    email: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('email') }})
    full_name: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('full_name') }})
    password: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('password') }})
    
